tasks/main.yml

```diff
@@ -5,14 +5,14 @@
 - name: test to see if selinux is running
   command: getenforce
   register: sestatus
-  changed_when: false
-  ignore_errors: true
+  failed_when: false
+  always_run: true
```

Comment:
yep, looks like this is the only option for making dry_run more accurate (if this is the goal of always_run). If I get everything right